src/main/java/Duke.java

```diff
@@ -49,52 +49,118 @@ public static void main(String[] args) {
             // Read inputs
             System.out.println("\tCall out a smurf to do a job for you!");
             Scanner in = new Scanner(System.in);
-            String command;
-            command = in.nextLine();
+            String userInput;
+            userInput = in.nextLine();
+            String[] userInputSubstrings = userInput.strip().split(" ", 2);
+            String command = userInputSubstrings[0];
```

Comment:
Perhaps a more meaningful name could be used for this?

Reply:
Do you have any suggestions?